digiwf-apps/packages/apps/digiwf-tasklist/src/components/task/TaskList.vue

```diff
@@ -154,7 +177,71 @@ export default class TaskList extends Vue {
       return [];
     }
 
-    return this.tasks.filter(task => JSON.stringify(Object.values(task)).toLocaleLowerCase().includes(this.syncedFilter.toLocaleLowerCase()));
+    return this.tasks.filter((task) =>
+      JSON.stringify(Object.values(task))
+        .toLocaleLowerCase()
+        .includes(this.syncedFilter.toLocaleLowerCase())
+    );
+  }
+
+  get isFilterPersistent(): boolean {
+    if (
+      !this.syncedFilter ||
+      this.syncedFilter.length == 0 ||
+      !this.persistentFilters ||
+      this.persistentFilters!.length == 0
+    ) {
+      return false;
+    }
+    return (
+      this.persistentFilters!.find(
+        (fl: FilterTO) => fl.filterString == this.syncedFilter
+      ) != undefined
+    );
+  }
+
+  async savePersistentFilter() {
+    const request: SaveFilterTO = {
+      pageId: this.pageId,
+      filterString: this.syncedFilter,
+    }
+    try {
+      const cfg = ApiConfig.getAxiosConfig(FetchUtils.getPUTConfig({}));
+      await FilterRestControllerApiFactory(cfg).saveFilter(request);
```

Comment:
```suggestion
    const persistentFilter: SaveFilterTO = {
      pageId: this.pageId,
      filterString: this.syncedFilter,
    }
    try {
      const cfg = ApiConfig.getAxiosConfig(FetchUtils.getPUTConfig({}));
      await FilterRestControllerApiFactory(cfg).saveFilter(persistentFilter);
```

It is a bit misleading that the variable `request` is a `SaveFilterTO` object and not a http request. I would rename it to e.g. `persistentFilter`